src/Playwright/Helpers/ChannelHelpers.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.Playwright.Transport;
+using Microsoft.Playwright.Transport.Channels;
+
+#nullable enable
+#pragma warning disable IDE0018 // Inline variable declaration will cause issues due to Roslyn design: https://github.com/dotnet/roslyn/issues/54711
+
+namespace Microsoft.Playwright.Helpers
+{
+    internal static class ChannelHelpers
+    {
+        internal static string? GetString(this JsonElement? element, string name, bool optional)
```

Comment:
@pavelfeldman your suggestion fit better now that I've refactored a bunch of the other calls... it feels more natural to say `Get`...